docker-compose/docker-compose.yml

```diff
@@ -64,21 +66,24 @@ services:
       INDEXER_MEMORY_LIMIT: "3"
       SHOW_TESTNET_LABEL: true
       SUPPORTED_CHAINS: '[]'
-      LOGO: /images/omni_logo.svg
+      LOGO: /images/omni_logo_long.svg
       APPS_MENU: false
-      FOOTER_LOGO: /images/omni_logo.svg
+      FOOTER_LOGO: /images/omni_logo_long.svg
       FOOTER_CHAT_LINK: https://discord.com/invite/bKNXmaX9VD
       FOOTER_FORUM_LINK_ENABLED: false
       FOOTER_TELEGRAM_LINK_ENABLED: true
       FOOTER_TELEGRAM_LINK: https://t.me/OmniFDN
       FOOTER_GITHUB_LINK: https://github.com/omni-network/blockscout
       FOOTER_LINK_TO_OTHER_EXPLORERS: false
       NEXT_PUBLIC_FOOTER_TWITTER_LINK: https://twitter.com/OmniFDN
+
     ports:
-      - 4000:4000
+       - 4000:4000
```

Comment:
github shows some space here, which can be a problem here?
also bunch of new lines which should not be impactful